.byebug_history

```diff
@@ -0,0 +1,91 @@
+continue
```

Comment:
@bapu-nexl you might want to remove this file and add it to .gitignore

Reply:
i will add this file and the coverage files here.